plugin/index.php

```diff
@@ -19,13 +23,26 @@ function gutenberg_menu() {
 	);
 }
 
+// Hook.
+add_action( 'admin_enqueue_scripts', 'gutenberg_scripts_and_styles' );
```

Comment:
>Having function and file level DocBlocks right after one and another makes less sense. 

In what way does it make less sense?

The relative positioning doesn't really impact much since it works in either case. My point in bringing it up was in whether it may be at odds with expectations other contributors would have coming in being familiar with differing patterns found more commonly in other plugins and core code.